### DISCUSSION TITLE
Information about canva

### DISCUSSION THREAD

**Discussion by ghost** [community/community#31577] [Accessibility]
I'm working on freelance marketplace. i need a canva pro version then sign up github student pack but github don't show me canva. Anyone please help me and solve my problem.
Thanks for your kind response in advance.

**Comment by toufiqul-hasan** [comment]
The Canva benefit is no longer offered in the GitHub Student Developer Pack.
For students who are already on the 12 month benefit, you will continue to have access for the remainder of your 12 month offer period & can contact Canva's support team with any related questions.
